hardware/VirtualThermostat.cpp

```diff
@@ -0,0 +1,682 @@
+#include "stdafx.h"
+#include "../main/Helper.h"
+#include "../main/Logger.h"
+#include "../main/SQLHelper.h"
+#include "../main/RFXtrx.h"
+
+#include "VirtualThermostat.h"
+
+#include "../main/localtime_r.h"
+#include "../main/mainworker.h"
+#include "../notifications/NotificationHelper.h"
+#include <stdarg.h>     
+
+std::string AVALAIBLE_MODE = "Eco,Conf,Frost,Off";
+
+std::string GetDeviceValue(const char* FieldName, const char* Idx)
+{
```

Comment:
Seems this function is not used as you made a new function in SQLHelper.cpp

Reply:
Ok!done delete